.github/workflows/lint-pending-strings.yml

```
@@ -20,6 +23,9 @@ jobs:
       - name: Install Python dependencies
         run: |
           pip install -r privaterelay/locales/.github/requirements.txt
-      - name: Lint reference
+      - name: Lint frontend pending translations
         run: |
           moz-fluent-lint frontend --config privaterelay/locales/.github/linter_config.yml
+      - name: Lint backend pending translations
+        run: |
+          moz-fluent-lint privaterelay/pending_locales
```

Shouldn't we add this to make sure we're adhering to the same rules as applied in the l10n repository? Or was there a reason you left it out?

```suggestion
          moz-fluent-lint privaterelay/pending_locales --config privaterelay/locales/.github/linter_config.yml
```

---

Good catch!